cmd/vic-machine/create/create.go

```diff
@@ -943,6 +943,16 @@ func (c *Create) Run(cliContext *cli.Context) (err error) {
 		log.SetLevel(log.DebugLevel)
 		trace.Logger.Level = log.DebugLevel
 	}
+
+	// urfave/cli will print out exit in error handling, so no more information in main method can be printed out.
+	defer func() {
```

Comment:
I thought about this one, but it also means we have to copy defer function everywhere. But as it can reduce copy/paste lines, I gonna use it.